weaver/samples/corda/corda-simple-application/clients/src/main/kotlin/com/cordaSimpleApplication/client/AssetTransferManager.kt

```diff
@@ -344,6 +345,7 @@ class ClaimRemoteAssetCommand : CliktCommand(name="claim-remote-asset", help = "
                 // SDK function claimPledgeFungibleAsset and Interop application for both the "Simple Asset" and
                 // the "Fungible house token" corDapps, we pass the Identity of the party submitting the claim here.
                 val thisParty: Party = rpc.proxy.nodeInfo().legalIdentities.get(0)
+                val issuer: Party = rpc.proxy.wellKnownPartyFromX500Name(CordaX500Name.parse("O=PartyA,L=London,C=GB"))!!
```

Comment:
Can you use a constant rather than hardcode a value in the function call?

Reply:
Done.